extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzSchedulerImpl.java

```diff
@@ -107,7 +107,6 @@
  * name of this class can be stored in a Quartz table in the database. See https://github.com/quarkusio/quarkus/issues/29177
  * for more information.
  */
-@Typed({ QuartzScheduler.class, Scheduler.class })
```

Comment:
Just out of curiosity, why was this `@Typed`?

Reply:
Hm, I guess that we did not want users to inject this concrete implementation type. And it actually make sense. I will revert this change - it's a residual from the previous iteration.